saleor/graphql/checkout/dataloaders.py

```diff
@@ -192,7 +192,7 @@ def batch_load(self, keys):
         for checkout in checkouts:
             key = (checkout.user_id, checkout.channel_slug)
             checkout_by_user_and_channel_map[key].append(checkout)
-        return [checkout_by_user_and_channel_map.get(key) for key in keys]
+        return [checkout_by_user_and_channel_map.get(key, []) for key in keys]
```

Comment:
It's `defaultdict(list)` that means `[]` is default value if key do not exists. 
```suggestion
        return [checkout_by_user_and_channel_map.get(key) for key in keys]
```